example/mini-express-app/server.js

```diff
@@ -0,0 +1,119 @@
+const Express = require('express')
+const BodyParser = require('body-parser')
+const Url = require('url')
+
+const Seneca = require('seneca')
+
+const StytchProvider = require('../../')
+
+
+const start = async () => {
```

Comment:
refactor this a little:
```
async function start() {
  const seneca = await makeSeneca() // do all the seneca stuff
  const app = makeExpress(seneca) // do all the express stuff
}
```

Reply:
this separates out seneca from the web framework, which is the approach we want to recommend in general, so lets provide an example here